src/main/java/com/mineinabyss/mobzy/mobs/MobDrop.kt

```diff
@@ -1,77 +1,48 @@
 package com.mineinabyss.mobzy.mobs
 
-import com.mineinabyss.idofront.items.editItemMeta
-import org.bukkit.Bukkit
-import org.bukkit.Material
-import org.bukkit.configuration.serialization.ConfigurationSerializable
+import com.mineinabyss.idofront.messaging.color
+import com.mineinabyss.idofront.recpies.addFurnaceRecipe
+import com.mineinabyss.idofront.serialization.SerializableItemStack
+import com.mineinabyss.mobzy.mobzy
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import org.bukkit.ChatColor
 import org.bukkit.inventory.ItemStack
 import kotlin.math.roundToInt
 import kotlin.random.Random
 
-data class MobDrop(val item: ItemStack,
-                   val minAmount: Int = 1,
-                   val maxAmount: Int = 1,
-                   val dropChance: Double = 1.0) : ConfigurationSerializable {
+@Serializable
+data class MobDrop(
+        val item: SerializableItemStack,
+        @SerialName("cooked-item") val cooked: SerializableItemStack? = null,
```

Comment:
json style

Reply:
Is json camelCased? I wonder if it makes sense to do that given that I can't find anything specific for yaml, though this is the way spigot serializes items.